src/admin.c

```diff
@@ -867,6 +867,28 @@ static bool admin_show_mem(PgSocket *admin, const char *arg)
 	return true;
 }
 
+/* Command: SHOW STATE */
+static bool admin_show_state(PgSocket *admin, const char *arg)
+{
+	PktBuf *buf;
+
+	buf = pktbuf_dynamic(64);
+	if (!buf) {
+		admin_error(admin, "no mem");
+		return true;
+	}
+
+	pktbuf_write_RowDescription(buf, "ss", "key", "value");
+
+	pktbuf_write_DataRow(buf, "ss", "active", (cf_pause_mode == P_NONE) ? "yes" : "no");
+	pktbuf_write_DataRow(buf, "ss", "pause", (cf_pause_mode == P_PAUSE) ? "yes" : "no");
+	pktbuf_write_DataRow(buf, "ss", "suspend", (cf_pause_mode == P_SUSPEND) ? "yes" : "no");
```

Comment:
Maybe like this to match the `paused` column of `SHOW DATABASES`
```suggestion
	pktbuf_write_DataRow(buf, "ss", "paused", (cf_pause_mode == P_PAUSE) ? "yes" : "no");
	pktbuf_write_DataRow(buf, "ss", "suspended", (cf_pause_mode == P_SUSPEND) ? "yes" : "no");
```

Reply:
Thanks for your review. I applied your suggestions. I will merge it soon.